extensions/amp-ad/0.1/utils.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @const {!Object} */
+export const ValidAdContainerTypes = {
```

Comment:
ValidAdContainerTypes is a misnomer. These are specifically ad containers that are ineligible for multisize, maybe something like MultisizeIneligibleAdContainerTypes (or something shorter/cleaner than that)